packages/levenshtein/levenshtein.1.1.0/opam

```diff
@@ -0,0 +1,17 @@
+opam-version: "1"
+maintainer: "jun.furuse@gmail.com"
+build: [
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"]
+  ["ocaml" "setup.ml" "-install"]
+]
+remove: [
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-uninstall"]
+]
+depends: [ 
+  "ocamlfind" 
+  "omake"
+  "pa_ounit" {>="109.53.02"}
```

Comment:
Isn't this a test dependency? You could specify it as `{ test & >= "109.53.02" }` and bump `opam-version` to `"1.2"`.